airbyte-integrations/connectors/source-sendgrid/integration_tests/no_spam_reports_configured_catalog.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please revert your changes on this file. It should not have the `messages` stream, this is why the test are not passing.